cns/NetworkContainerContract.go

```diff
@@ -77,6 +77,12 @@ const (
 	MultiTenantCRD = "MultiTenantCRD"
 )
 
+// Address Types
+const (
+	Default   = "Default"
+	Secondary = "Secondary"
+)
```

Comment:
suggest making this an "enum" like
```
type AddressType string

const (
    AddressTypeDefault = "Default"
    AddressTypeSecondary = "Secondary"
)

Reply:
will this be replaced by `nicType` instead of `addressType` @jaer-tsun ?